tests/dart/DartLintBearTest.py

```diff
@@ -30,3 +37,23 @@
                                      valid_files=(good_file,),
                                      invalid_files=(bad_file,),
                                      tempfile_kwargs={"suffix": ".dart"})
+
+
+@skipIf(which('dartanalyzer') is None, 'dartanalyzer is not installed')
+class DartLintBearConfigTest(LocalBearTestHelper):
+
+    def test_config_failure_use_spaces(self):
+        section = Section("name")
+        section.append(Setting('use_spaces', False))
+        bear = DartLintBear(section, Queue())
+
+        with pytest.raises(AssertionError):
```

Comment:
Why not use unittest's assertRaises ? https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertRaises

It has the same functionality (The reason I mention is because it's nicer to use the builtin unittest as much as possible)